rgw/v2/tests/s3_swift/test_bucket_lifecycle_object_expiration.py

```diff
@@ -147,6 +147,11 @@ def test_exec(config, ssh_con):
                 )
             )
             prefix = prefix if prefix else ["dummy1"]
+            if (
+                config.test_ops["enable_versioning"]
```

Comment:
@viduship , could you use "enable_resharding" intead of "enable_versioning" as discussed